src/l1-resources/search-index/README.md

```diff
@@ -2,14 +2,14 @@
 
 The official [MongoDB Atlas](https://www.mongodb.com/) AWS CDK resource for Node.js.
```

Comment:
Why are we making our notes specific to Node.js for AWS CDK? unless there is strong reason to do so suggest remove given users will be able to leverage this CDK in other languages

Reply:
i didn't change that, i suppose we have for all resources